Kudu.Core.Test/Infrastructure/EnvironmentFacts.cs

```diff
@@ -125,7 +125,8 @@ public void SSHKeyPathCreatesDirectoryIfItDoesNotExist()
             string sshKeyPath = null,
             string scriptPath = null,
             string nodeModulesPath = null,
-            string dataPath = null)
+            string dataPath = null,
+            string binPath = null)
```

Comment:
Can you explain a bit why binPath knowledge is needed now - but not before?   what scenario etc?